scripts/build/package.js

```diff
@@ -79,8 +83,24 @@ data.icons.forEach(icon => {
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
   minifyAndWrite(jsFilepath, `module.exports=${iconToObject(icon)};`);
+
+  // add export to the barrel file
+  const firstLetter = icon.slug[0].toUpperCase();
+  const rest = icon.slug.slice(1);
+  const iconExportName = `icon${firstLetter}${rest}`;
+  const iconJsFilepath = `./icons/${filename}.js`;
+
+  iconsBarrelJs.push(`${iconExportName}:require("${iconJsFilepath}"),`);
+  iconsBarrelMjs.push(`export{default as ${iconExportName}}from"${iconJsFilepath}";`);
 });
 
 // write our generic index.js
 const rawIndexJs = util.format(indexTemplate, icons.map(iconToKeyValue).join(','));
 minifyAndWrite(indexFile, rawIndexJs);
+
+// write to our file containing the exports of all icons
+const iconsJs = `module.exports={${iconsBarrelJs.join('')}};`;
+fs.writeFileSync(iconsJsFile, iconsJs);
+
+const iconsMjs = iconsBarrelMjs.join('');
+fs.writeFileSync(iconsMjsFile, iconsMjs);
```

Comment:
For the sake of consistency, and to be absolutely 100% sure this is as compact as possible, I think we should be using `minifyAndWrite` here. (also some minor other suggestions)

```suggestion
// write our file containing the exports of all icons in CommonJS ...
const rawIconsJs = `module.exports={${iconsBarrelJs.join("")}};`;
minifyAndWrite(rawIconsJs, iconsJs);
// and ESM
const rawIconsMjs = iconsBarrelMjs.join("");
minifyAndWrite(iconsMjsFile, rawIconsMjs);
```